test/PerpetualPositionRouter.t.sol

```diff
@@ -221,22 +221,35 @@ contract Fallback is PositionRouterTest {
   }
 
   function testFork_FailedCallWhenExtraCalldataArgument() public {
+    delegateApproval.approve(vethPositionRouterAddr, 1);
     (bool ok,) = payable(vethPositionRouterAddr).call(
-      abi.encodePacked(uint8(4), uint160(0), uint32(type(uint32).max), uint96(0), uint96(100))
+      abi.encodePacked(uint8(4), uint160(0), uint32(block.timestamp), uint96(0), uint96(100))
     );
     assertTrue(!ok);
   }
 
   function testFork_FailedClosePositionCallWithWrongArguments() public {
+    delegateApproval.approve(vethPositionRouterAddr, 1);
     (bool ok,) = payable(vethPositionRouterAddr).call(
-      abi.encodePacked(uint8(5), uint160(0), uint32(type(uint32).max), uint96(1 ether), uint96(0))
+      abi.encodePacked(uint8(5), uint160(0), uint32(block.timestamp), uint96(1 ether), uint96(0))
     );
     assertTrue(!ok);
   }
 
   function testFork_FailedFallbackWithZeroFuncId() public {
+    delegateApproval.approve(vethPositionRouterAddr, 1);
     (bool ok,) = payable(vethPositionRouterAddr).call(
-      abi.encodePacked(uint8(0), uint160(0), uint32(type(uint32).max), uint96(1 ether), uint96(0))
+      abi.encodePacked(uint8(0), uint160(0), uint32(block.timestamp), uint96(1 ether), uint96(0))
+    );
+    assertTrue(!ok);
```

Comment:
Looks like we have this in a few places
```suggestion
    assertFalse(ok);
```

Reply:
https://github.com/ScopeLift/optimizoors-perpetual-protocol/pull/27/commits/f74d1ce6801af48d966d018187206af5c0839a69